terra-aou-rstudio/rstudio/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e # catches errors
+
+# Restrict uploads to 4096 kbit/s (0.512 MB/s) and unlimited download
+tc qdisc add dev eth0 root tbf rate 4096kbit latency 50ms burst 1540000
+
+# Init must be run as root and will internally handle mapping the end-user
+# to UID and GID mapping using S6-overlay
+# https://github.com/just-containers/s6-overlay
+# using S6
+# https://skarnet.org/software/s6/overview.html
+/init
```

Comment:
```suggestion
su-exec root /init
```
Sorry, this is what I meant - every process called from the entrypoint script should be prepended with `su-exec root`, so that the entrypoint script exits after forking these processes.

Reply:
```
Tasks:   7 total,   1 running,   6 sleeping,   0 stopped,   0 zombie
%Cpu(s):  0.2 us,  0.0 sy,  0.0 ni, 99.7 id,  0.2 wa,  0.0 hi,  0.0 si,  0.0 st
MiB Mem :   7946.7 total,   1589.5 free,    481.1 used,   5876.1 buff/cache
MiB Swap:      0.0 total,      0.0 free,      0.0 used.   7159.1 avail Mem 

    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND                                          
    313 rstudio   20   0  923360 149224  79628 S   0.3   1.8   0:00.86 rsession                                         
      1 root      20   0     204      4      0 S   0.0   0.0   0:00.02 s6-svscan                                        
     37 root      20   0     204      4      0 S   0.0   0.0   0:00.00 s6-supervise                                     
    267 root      20   0     204      4      0 S   0.0   0.0   0:00.00 s6-supervise                                     
    270 rstudio+  20   0  258308  17800  15372 S   0.0   0.2   0:00.15 rserver                                          
    339 rstudio   20   0    8668   5420   3736 S   0.0   0.1   0:00.01 bash                                             
    343 rstudio   20   0   10376   3916   3344 R   0.0   0.0   0:00.00 top                                              
```